regolith/helpers/l_generalhelper.py

```diff
@@ -0,0 +1,78 @@
+"""Helper for listing filtered data from collections in the database.
+"""
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    search_collection,
+    collection_str,
+)
+HELPER_TARGET = "lister"
+
+def subparser(subpi):
+    subpi.add_argument(
+        "coll",
+        help='collection where the lister is going to run.')
+    subpi.add_argument(
+        "-f","--kv_filter",
+        nargs="+",
+        help='search the given collection by key element pairs')
```

Comment:
this needs an example or it will be too hard for people to know what to do here.  If you search around you should see a nice help message for this in other listers, written by Michael.

Also, let's decide if we are going to put periods at the end of help messages or not, and make it all consistent.  I think yes, have a period.  It's better English.

Reply:
Does it look better?